stdlib/LinearAlgebra/test/symmetric.jl

```diff
@@ -663,4 +663,19 @@ end
     @test LinearAlgebra.hermitian_type(Int) == Int
 end
 
+@testset "sqrt(nearly semidefinite)" begin
+    let A = [0.9999999999999998 4.649058915617843e-16 -1.3149405273715513e-16 9.9959579317056e-17; -8.326672684688674e-16 1.0000000000000004 2.9280733590254494e-16 -2.9993900031619594e-16; 9.43689570931383e-16 -1.339206523454095e-15 1.0000000000000007 -8.550505126287743e-16; -6.245004513516506e-16 -2.0122792321330962e-16 1.183061278035052e-16 1.0000000000000002],
+        B = [0.09648289218436859 0.023497875751503007 0.0 0.0; 0.023497875751503007 0.045787575150300804 0.0 0.0; 0.0 0.0 0.0 0.0; 0.0 0.0 0.0 0.0],
+        C = Symmetric(A*B*A'), # semidefinite up to roundoff
+        Csqrt = sqrt(C)
+        @test Csqrt isa Symmetric{Float64}
+        @test Csqrt*Csqrt ≈ C rtol=1e-15
```

Comment:
The test failure on AArch64 is genuine. Since `C` has elements that are very small, I guess you'd have to use an absolute tolerance here?

Reply:
You don't need an absolute tolerance just because elements are small — you only need an absolute tolerance if the *whole array* is going to zero.  Remember that the test is `‖M - M̃‖ ≤ rtol*‖M‖`

1e-15 is probably just too aggressive (i.e. too close to machine precision); I'll change it to 1e-15.